website/content/configuration/_index.md

```diff
@@ -36,44 +61,56 @@ Layer 2 mode does not require the IPs to be bound to the network interfaces
 of your worker nodes. It works by responding to ARP requests on your local
 network directly, to give the machine's MAC address to clients.
 
+In order to advertise the IP coming from an `IPAddressPool`, a `L2Advertisement`
```

Comment:
super nit: should be **an** `L2Advertisement` (reading this as `el-two`) but your choice not sure what's better :sweat_smile: